frontend/tests/page-object/base.page.ts

```diff
@@ -0,0 +1,8 @@
+import type { Page } from "@playwright/test";
+
+export const BasePage = {
+    goto: async (page: Page, endpoint: string) => {
+        await page.goto("https://v2-staging-mainnet.beaconcha.in" + endpoint);
```

Comment:
I guess this should suffice, if we set the `base url` in the config
```suggestion
        await page.goto(endpoint);
```